lib/Controller/ApiController.php

```diff
@@ -206,6 +208,54 @@ public function export(array $migrators): DataResponse {
 			throw new OCSException('User migration operation already queued');
 		}
 
+		return $user;
+	}
+
+	/**
+	 * @NoAdminRequired
+	 * @NoSubAdminRequired
+	 *
+	 * @throws OCSException
+	 */
+	public function exportable(?array $migrators): DataResponse {
+		$user = $this->checkJobAndGetUser();
+
+		if (!is_null($migrators)) {
+			$this->checkMigrators($migrators);
+		}
+
+		try {
+			$size = $this->migrationService->estimateExportSize($user, $migrators);
+			// Convert to MiB and round to one significant digit after the decimal point
+			$roundedSize = round($size / 1024, 1);
+		} catch (UserMigrationException $e) {
+			throw new OCSException($e->getMessage());
+		}
+
+		try {
+			$this->migrationService->checkExportability($user, $migrators);
+		} catch (NotExportableException $e) {
+			$warning = $e->getMessage();
```

Comment:
Why is that not a `throw new OCSException($e->getMessage());`?

Reply:
I would also maybe reverse the order, test checkExportability first, to not call size estimation computation if there is another problem.

Reply:
We don't throw here because the purpose of this endpoint is to always return the size estimate and a warning if there is one in the same response, throwing would block that